src/main/kotlin/org/jitsi/videobridge/octo/OctoEndpoint.kt

```diff
@@ -72,6 +73,14 @@ class OctoEndpoint(
         // single OctoEndpoints instance.
     }
 
+    override fun setVideoConstraints(newVideoConstraints: MutableMap<String, VideoConstraints>?) {
+        TODO("Not yet implemented")
```

Comment:
If we want this to be a no-op, we should remove the `TODO` as it'll throw an exception if called. (and below)

Reply:
I don't think octo video constraint propagation will go through this so I'll make it a NO-OP.